seatunnel-e2e/seatunnel-connector-v2-e2e/connector-starrocks-e2e/src/test/resources/ddl/add_columns.sql

```diff
@@ -0,0 +1,83 @@
+--
```

Comment:
ok,thanks!

Reply:
@dailai starrocks ddl does not support default values, while the alter in add_column has default values. It may still be necessary to use the add_column in Starrocks e2e